lib/localeapp/cli/install.rb

```diff
@@ -66,12 +66,9 @@ def validate_key
           end
         end
 
-        def check_default_locale
+        def print_default_locale
           localeapp_default_code = project_data['default_locale']['code']
           @output.puts "Default Locale: #{localeapp_default_code} (#{project_data['default_locale']['name']})"
```

Comment:
I agree, and made the change, but I think we can refactor many `#puts`
calls into one (for both message), let me know if that's OK.